user_docs/en/userGuide.t2t

```diff
@@ -1576,6 +1576,16 @@ It has the following options:
 - Beep and speak: This option tells NVDA to both beep and speak when a progress bar updates.
 -
 
+==== Progress bar speech percentage Interval ====[ObjectPresentationReportProgressBarSpeechPercentageInterval]
+This setting allows you to set after which percentage change the new progress bar state will be read by the synthesizer.

```

Comment:
Could you add examples? For example:

`For example, when the value of this option is set to 10, NVDA will announce the value of the progress bar every time the percentage increased with 10%`